client/bindings/wasm/build_scripts/lints.js

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
Just curious, did you write this file or is this a common thing to add when dealing with wasm/rust ?

Reply:
We have the same lints in the identity.rs WASM bindings [the file can be found here](https://github.com/iotaledger/identity.rs/blob/main/bindings/wasm/build/lints.js) . The `lintPtrNullWithoutFree` was added in  [this PR](https://github.com/iotaledger/identity.rs/pull/694) which contains a good explanation why it was added.  Since this is only relevant when passing the `--weak-refs` flag to `wasm-bindgen` CLI which is not the default [see this section in the wasm-bindgen guide](https://rustwasm.github.io/wasm-bindgen/reference/weak-references.html), I don't expect this lint to be widely used as of now. WASM is still somewhat niche and perhaps still arguably more common in high performance browser applications rather than exported TS/JS libraries. For the latter weak references is rather essential for a decent UX as one has to manually call `free` on exported structs otherwise. 

The second lint was added in a more extensive PR to our project. I have not seen occurrences of that in the "wild" either, but I might just not have tried hard enough.

Reply:
Thanks for the explanation 👍🏻